src/novaprobe.py

```diff
@@ -143,6 +144,8 @@ class ArgHolder(object):
     parser.add_argument('-t', dest='timeout', type=int, nargs='?', default=120)
     parser.add_argument('--capath', dest='capath', nargs='?', default='/etc/grid-security/certificates')
     parser.add_argument('--appdb-image', dest='appdb_img', nargs='?')
+    parser.add_argument('--protocol', dest='protocol', default='oidc', nargs='?')
+    parser.add_argument('--indeitity-provider', dest='identity_provider', default='egi.eu', nargs='?')
```

Comment:
There is a typo in the argument, could you please fix it?

Reply:
ups, fixed